migrations/20170125115201_create_document_categories.js

```diff
@@ -0,0 +1,173 @@
+'use strict';
+
+const documentCategories = [
+  {
+    id: 19,
```

Comment:
I really don't like the idea of hardcoding IDs into migrations. Isn't there any other way of doing it? Maybe we can further match explorer and API categories by name+group pairs, which are unique.